AEPServices/Sources/storage/FileSystemNamedCollection.swift

```diff
@@ -92,13 +92,16 @@ class FileSystemNamedCollection: NamedCollectionProcessing {
             return nil
         }
 
-        if let storageData = try? Data(contentsOf: fileUrl) {
-            if let jsonResult = try? JSONSerialization.jsonObject(with: storageData) as? [String: Any] {
-                return jsonResult
-            }
+        guard let storageData = try? Data(contentsOf: fileUrl) else {
+            return nil
```

Comment:
should we provide a message here indicating we were unable to retrieve the file contents?

Reply:
This is causing a lot of false warnings during a new install.